python/ray/train/examples/pipeline_parallelism_example.py

```diff
@@ -0,0 +1,352 @@
+import argparse
```

Comment:
Should we also add this to here https://docs.ray.io/en/latest/train/examples.html?